### ISSUE TITLE
Train CIFAR10 models (with e.g. Google Colab)

### ISSUE THREAD

**Issue by travela** [privML/privacy-evaluator#28]
Definition of Done for our model: one trained model in PyTorch, one in Tensorflow, demonstrate accuracy on test data for the two different classes --> should be a reasonable accuracy (80-90)

**Comment by TobiasSchalau** [comment]
model construction for Pytorch is ready
tofo: create model with Tesorflow and add a test for accuracy

**Comment by yzchyx** [comment]
For both PyTorch and TensorFlow we apply a pretrained ResNet50 model and freeze all but the last layers. The results are good (80~90%) for binary classification. [Source code](https://github.com/yzchyx/privacy-evaluator/tree/feat/28-train-cifar/demo):
#### Results
|Dataset|Torch+ResNet50|TF+Resnet50|
|:-:|-:|-:|
|{0: 5000, 1: 5000}|88.30%|92.40%|
|{0: 5000, 1: 4000}|87.65%|92.30%|
|{0: 5000, 1: 3000}|87.70%|92.15%|
|{0: 5000, 1: 2000}|87.85%|90.95%|
|{0: 5000, 1: 1000}|85.15%|89.55%|
|{0: 5000, 1: 500} |83.65%|87.75%|
|{0: 3000, 1: 1000, 2: 500}|72.83%|79.80%|
